src/apps/frontend/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface BreadcrumbProps {
+  pageName: string;
+}
+const Breadcrumb: React.FC<BreadcrumbProps> = ({ pageName }) => (
```

Comment:
add new line above.

Reply:
Is this implementation coming from Tailwind admin? If yes, then it may be ok, but as I see it has few limitation:

1. Breadcrumb can have multiple levels, this component only supports 1
2. Ideally there should be breadcrumb and breadcrumb items
3. `pageName` seems like a concept related to page, this should be more likely breadcrumbItem or something like that
4. Is it `bread-crumb` or `breadcrumb` i.e. are those 2 words or one word?